lms/djangoapps/courseware/tests/test_self_paced_overrides.py

```diff
@@ -118,31 +118,31 @@ def test_course_access_to_beta_users(self):
         self.assertTrue(has_access(beta_tester, 'load', self_paced_section, self_paced_course.id))
 
     @patch.dict('courseware.access.settings.FEATURES', {'DISABLE_START_DATES': False})
-    def test_instructor_paced_discussion_module_visibility(self):
+    def test_instructor_paced_discussion_xblock_visibility(self):
         """
-        Verify that discussion modules scheduled for release in the future are
+        Verify that discussion xblock scheduled for release in the future are
         not visible to students in an instructor-paced course.
         """
         course, section = self.setup_course(start=self.now, self_paced=False)
-        self.create_discussion_modules(section)
+        self.create_discussion_xblocks(section)
 
         # Only the released module should be visible when the course is instructor-paced.
-        modules = get_accessible_discussion_modules(course, self.non_staff_user)
+        modules = get_accessible_discussion_xblocks(course, self.non_staff_user)
         self.assertTrue(
             all(module.display_name == 'released' for module in modules)
         )
 
     @patch.dict('courseware.access.settings.FEATURES', {'DISABLE_START_DATES': False})
-    def test_self_paced_discussion_module_visibility(self):
+    def test_self_paced_discussion_xblock_visibility(self):
         """
-        Regression test. Verify that discussion modules scheduled for release
+        Regression test. Verify that discussion xblock scheduled for release
         in the future are visible to students in a self-paced course.
         """
         course, section = self.setup_course(start=self.now, self_paced=True)
-        self.create_discussion_modules(section)
+        self.create_discussion_xblocks(section)
 
         # The scheduled module should be visible when the course is self-paced.
```

Comment:
Addressed in [d46d2f9](https://github.com/edx/edx-platform/pull/12582/commits/d46d2f99c65124a3f0578c4c0163035703937e7e).